src/components/MenuSection/MenuSection.module.css

```diff
@@ -0,0 +1,79 @@
+.menuSection {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  padding: 140px 120px;
```

Comment:
Looks as a huge horizontal paddings. May be we could define smaller container?

Reply:
Fixed